etl/steps/data/grapher/who/2023-11-01/who_statins.py

```diff
@@ -0,0 +1,32 @@
+"""Load a garden dataset and create a grapher dataset."""
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load garden dataset.
+    ds_garden = paths.load_dataset("who_statins")
+
+    # Read table from garden dataset.
+    tb = ds_garden["who_statins"]
+
+    #
+    # Process data.
```

Comment:
Maybe delete this to be extra neat :)

Reply:
thanks for taking a look at this so quickly @spoonerf  :)